### DISCUSSION TITLE
Splitter: Uncaught ReferenceError: horizontal is not defined

### DISCUSSION THREAD

**Discussion by Matthew404-fail** [primefaces/community#1112] [PrimeReact] [answered]
I tried to use a `Splitter`, but when I tried to resize it, it caused an error - "Uncaught ReferenceError: horizontal is not defined". 
Then I decided to run its demo, but got exactly the same error.
I need your help, thank you in advance.
![image](https://github.com/primefaces/community/assets/66127533/9f0e8ab2-14d4-4a99-8702-f303e57fc834)
![image](https://github.com/primefaces/community/assets/66127533/0563cdff-b059-457a-8943-8496072b9617)

**Comment by melloware** [comment, marked as answer]
Already reported and fixed: https://github.com/primefaces/primereact/issues/5892

**Reply by Matthew404-fail** [reply]
Thanks a lot!
